### PR TITLE
Fixed #431 (add tutorial to Speak)

### LINKED ISSUE #431: Add a tutorial to the Speak activity
Add a tutorial to the Speak activity following tutorial 9 in Sugarizer development tutorial: https://github.com/llaske/sugarizer/blob/dev/docs/tutorial_step9.md

The tutorial:
* [ ] Should display a welcome message to explain the activity (could reuse the help message describing the activity in Sugarizer activities List view)
* [ ] Should explain usage of each button in the toolbar
* [ ] Should be localized
* [ ] Should respect the Sugarizer UI, as explained in tutorial

### PR DESCRIPTION
**Pull request by AmiMappy**
Added tutorial feature to the activity

### CODE REVIEW

File: activities/Speak.activity/lib/tutorial.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+			{
+				element: "",
+				orphan: true,
+				placement: "bottom",
+				title: l10n.get(TutoExplainTitle),
+				content:l10n.get(TutoExplainContent)
+			},
+			{
+				element: "#gamemode1-button",
+				title: "gamemode1",
```
Comment (llaske) [inline activities/Speak.activity/lib/tutorial.js]: All titles should be localized

File: activities/Speak.activity/js/activity.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-define(["sugar-web/activity/activity","activity/SpeakActivity","facepalette","speechpalette","languagepalette","activity/sax","activity/dom-js","activity/AIMLInterpreter","activity/Speech"], function (activity, speakActivity, facepalette, speechpalette, languagepalette) {
+define(["sugar-web/activity/activity","activity/SpeakActivity","facepalette","speechpalette","languagepalette","activity/sax","activity/dom-js","activity/AIMLInterpreter","activity/Speech", "tutorial", "webL10n"], function (activity, speakActivity, facepalette, speechpalette, languagepalette, tutorial, webL10n) {
```
Comment (llaske) [inline activities/Speak.activity/js/activity.js]: Look like you don't need to add webL10n.js

File: activities/Speak.activity/lib/tutorial.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+                        </div>\
+                        <div class='icon-tutorial-text'>"+l10n.get("TutoEnd")+"</div>\
+                    </div>\
+                </div>\
+            </div>",
+            storage: false,
+            backdrop: true,
+            steps: steps
+        });
+		tour.init();
```
Comment (llaske) [inline activities/Speak.activity/lib/tutorial.js]: Do not mix TAB indentation and white spaces indentation in the file

**Comment by AmiMappy** [comment]
Unfortunately, I still can't wrap my head around the other persistent issue.

**Comment by llaske** [comment]
Look like you've mixed two fixes: one for Speak and one for FoodChain. Each PR should concern one issue only

**Comment by llaske** [comment]
Fixed in https://github.com/llaske/sugarizer/pull/534

**Pull request closed without merging**